charts/internal/openstack-infra/templates/main.tf

```diff
@@ -5,7 +5,7 @@ provider "openstack" {
   region      = "{{ required "openstack.region is required" .Values.openstack.region }}"
   user_name   = var.USER_NAME
   password    = var.PASSWORD
-  insecure    = true
```

Comment:
Removing this probably means to provide a way to pass the CA certificate for non-globally trusted server certs, or?

Reply:
yes

Reply:
Hmm yes this was maybe to rushed and will not work in all openstack environments without providing the ability to pass the ca cert.

Reply:
Let's add `insecure=true` again and remove it with a subsequent PR where we also provide the ability to pass a proper CA cert to do the TLS. WDYT?

Reply:
Agreed, makes sense. Otherwise, it will break in environments not under our control.

Reply:
done